README.md

```diff
@@ -35,30 +40,36 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`EXPERIMENT_ID` : set the experiment ID, or leave unset for default experiment.
+`BIOMAGE_DATA_PATH`: where to get the experiment data to populate inframock's S3 and DynamoDB. If
+this is not set, it will default to `./data`.
 
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
-set to `true` by default, which means that InfraMock will try to use a locally running version of
-the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint.
-For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
-
-`MOCK_EXPERIMENT_DATA_PATH`: where to get the mocked data for upload to the local S3 from. If
-this is not set, it will default to the Github URL where the dataset used for the `worker` unit
-test is located.
+`POPULATE_MOCK`: whether localstack should be filled with the data sets found in `BIOMAGE_DATA_PATH`.
+For this to work, `CLUSTER_ENV` must be set to `development`, which is the default behavior.
 
 `AWS_DEFAULT_REGION`: the default mocked region for your infrastructure to be deployed under. If it's not set,
 it defaults to `eu-west-1`.
 
-`USE_LOCAL_DATA`: Flag to use local data or pull from Github. Set to `false` by defult. Set to `true` to use local data.
-
-Using local copy of data
+Adding custom data
 ---------------------
 
-Everytime Inframock is run, it will pull data from [](https://github.com/biomage-ltd/worker/tree/master/data/test). These files can be large. Inframock has `USE_LOCAL_DATA` environment variable path to command inframock to use local data. To use this feature :
+Inframock loads automatically the experiments found in the `./data` folder. The default experiment included is the same found in the worker repo [here](https://github.com/biomage-ltd/worker/blob/master/data/test/r.rds.gz). The expected format for loading data is the following:
+
```

Comment:
Now we have `data` in both the worker and inframock. It can happen that people update the datasets in the worker, then they forget to update the dataset in inframock and after testing, they "think" that testing locally has been completed with the new dataset, when it actually hasn't. Is the worker data under `.data` used for anything else now apart from unit tests? If yes, do you have any ideas on what can we do to prevent such situations?

Reply:
Never mind, I jsut read the readme in the `.data` folder, makes a lot of sense now :)